ember-cli-build.js

```diff
@@ -37,7 +37,6 @@ module.exports = function(defaults) {
             }],
         };
     }
-
```

Comment:
???

Reply:
This must have been from my mystery stash. Fixed now though